backend/lcfs/web/api/compliance_report/schema.py

```diff
@@ -47,12 +48,8 @@ class PortsEnum(str, Enum):
     DUAL = "Dual port"
 
 
-class CompliancePeriodSchema(BaseSchema):
-    compliance_period_id: int
-    description: str
-    effective_date: Optional[datetime] = None
-    expiration_date: Optional[datetime] = None
-    display_order: Optional[int] = None
+class CompliancePeriodSchema(CompliancePeriodBaseSchema):
```

Comment:
This is odd, do we still need CompliancePeriodSchema or can we just use the BaseSchema directly?

Reply:
I am updating to use the BaseSchema, with this approach the code did not have to change in the imports and references.